src/client/components/EditEventsModal/EditEventsModal.stories.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import EditEventsModal from './EditEventsModal';
+import ElasticLogo from '../StudyGroupLogo/Elastic.logo.svg';
+
+const studygroups = [
+  {
+    id: '01',
+    value: 'Elastic',
+  },
+  {
+    id: '02',
+    value: 'Zendesk',
+  },
+  {
+    id: '03',
+    value: 'Hack Your Future',
+  },
+];
+
+const EditEvents = ({ items }) => {
+  const [value, setValue] = useState('');
+  const emptyInput = value === '';
+  return (
+    <EditEventsModal
+      items={items}
+      value={value}
+      setValue={setValue}
+      heading="Edit Event (Time Slot)"
+      path={ElasticLogo}
+      title="Elastic - Week 46"
+      datetime="Wednesday | 11 Nov 20|10:00 CEST"
+      mentor="name"
+      link="www.hyf.org"
+      disabled={emptyInput}
+    />
+  );
+};
```

Comment:
You still have a component in story that is not needed. Why don't you move all this code under the `EditNotPastEvent` below?